src/components/atoms/SplitView/SplitView.tsx

```diff
@@ -158,6 +178,13 @@ const SplitView: FunctionComponent<SplitViewProps> = ({
     setDraggingLeftNav(false);
     setDraggingNavEdit(false);
     setDraggingEditRight(false);
+    dispatch(updateLeftWidth(leftWidth));
```

Comment:
perhaps just have one updatePaneConfiguration reducer method that takes an object with all values!?